src/go/k8s/pkg/resources/headless_service.go

```diff
@@ -22,7 +22,7 @@ import (
 	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 
-	redpandav1alpha1 "github.com/redpanda-data/redpanda/src/go/k8s/apis/redpanda/v1alpha1"
```

Comment:
There are a lot of files with this nit change, i suppose this is cleanup from the domain change, in the future let's split this out in another PR since it is fairly clean what the change is.